vnext/src/Libraries/Components/TextInput/TextInput.windows.js

```diff
@@ -67,9 +66,8 @@ if (Platform.OS === 'android') {
 // [Windows
 else if (Platform.OS === 'windows') {
   WindowsTextInput = requireNativeComponent('RCTTextInput');
-  WindowsTextInputCommands = codegenNativeCommands<
-    TextInputNativeCommands<HostComponent<any>>,
-  >({supportedCommands: ['focus', 'blur', 'setTextAndSelection']});
+  WindowsTextInputCommands = require('./WindowsTextInputNativeCommands')
```

Comment:
Where are these called from this file? (I'm on mobile so I can't check the rest of the file). I would have expected it to only be called from inside TextInputState

Reply:
Assigned to viewCommands on line 867, which is used for calling setTextAndSelection - I just wanted not to use one that has Android in its name.
```
let viewCommands: TextInputNativeCommands<HostComponent<any>>;
  if (AndroidTextInputCommands) {
    viewCommands = AndroidTextInputCommands;
  }
  // [Windows
  else if (WindowsTextInputCommands) {
    viewCommands = WindowsTextInputCommands;
  }
```
s

Reply:
Ah cool